crates/matrix-sdk/tests/integration/room/joined.rs

```diff
@@ -126,6 +126,28 @@ async fn test_leave_room() -> Result<(), anyhow::Error> {
     Ok(())
 }
 
+#[async_test]
+async fn test_leave_invited_room_also_forgets_it() -> Result<(), anyhow::Error> {
```

Comment:
Can we test that we _don't_ forget a room that we were joined?

Reply:
Done and force pushed the new test.